### ISSUE TITLE
Possible bug: backspace doesn't work in "log to custom url" option

### ISSUE THREAD

**Issue by fsarrio** [mendhak/gpslogger#291]
I’m using  the last version (56) on a Samsung Galaxy Express 2.

If I enter to modify the URL from the “Log to custom URL” option the backspace key doesn’t work. In other applications such mail, text editor,… works fine. Before the last update the backspace key did work. 

Thank you. You are doing a very good job with this application.

**Comment by mendhak** [comment]
Definitely one of the stranger bugs I've heard.  Are you using the native Google Keyboard?  If yes - it is possible that it's this known bug:

https://code.google.com/p/android/issues/detail?id=62306

https://code.google.com/p/android/issues/detail?id=42904

In other words, some dialogs that pops up with a textbox in it doesn't allow backspace; it only allows backspace for characters you entered in that session.

Can you also try highlighting the entire text in that textbox, then using space or some other letter to 'overwrite' it?

You can also try to prove or disprove this by trying another keyboard.  Change the input keyboard, then see if you can delete text.

**Comment by fsarrio** [comment]
How can i check if i am using the native Google Keyboard?
Yesterday I installed GPS Logger in three different smartphones (sony & samsung). I could delete alphanumeric characters, but backspace didn't work when found special chars (% : /) from variables and protocol (http://).
Finally I wrote the URL in another app, I copied the URL and pasted in GPS Logger.

Now, in the original phone I tried to select all text and write something, it worked fine, but same problem with backspace key and the new text.
